common_gaku_gemspec_mixin.rb

```diff
@@ -0,0 +1,10 @@
+def set_gaku_gemspec_shared(s)
+  s.platform    = Gem::Platform::RUBY
+  s.required_ruby_version = '~> 2.1.3'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.